content/en/references/ibd-system-ventilator-splitter-assembly-airflow.adoc

```diff
@@ -0,0 +1,12 @@
+---
+title: "Splitter Assembly UML Diagram"
+linkTitle: "Splitter Assembly UML Diagram"
+weight: 1
+description: >
+  UML diagram of the Project Tetra splitter assembly.
```

Comment:
I would re-title the description to _Interface control document for ventilator splitter assembly_